packages/taucmdr/cf/software/tau_installation.py

```diff
@@ -644,6 +641,17 @@ def get_shared_dir(tau_makefile):
         """
         return 'shared'.join(tau_makefile.rsplit('Makefile.tau', 1))
 
+                elif 'DWARFINC=' in line:
+                    if self.uses_libdwarf:
+                        libdwarf = self.dependencies['libdwarf']
+                        libdwarf_dir = line.split('=')[1].strip().strip("-I")
+                        if not os.path.isdir(libdwarf_dir):
+                            raise SoftwarePackageError("DWARFINC in '%s' is not a directory" % tau_makefile)
+                        if libdwarf.include_path != libdwarf_dir:
+                            LOGGER.debug("DWARFINC='%s' != '%s'", libdwarf_dir, libdwarf.include_path)
+                            raise SoftwarePackageError("DWARFINC in '%s' is not '%s'" %
+                                                        (tau_makefile, libdwarf.include_path))
+
```

Comment:
These lines are indented too far, which is causing an `IndentationError` during installation.